en/admin/config.rst

```diff
@@ -1777,6 +1782,38 @@ The following are parameters related to the query plan cache functionality. The
 
     **max_filter_pred_cache_entries** is a parameter used to specify the maximum number of filtered index expressions. The filtered index expressions are stored with them complied and can be immediately used in server. If it is not stored in cache, the process is required which filtered index expressions are fetched from database schema and interpreted.
 
+.. _query-cache-parameters:
+
+Query Cache-Related Parameters
+-----------------------------------
+
+The following are parameters related to the query cache functionality. The type and value range for each parameter are as follows:
+
++-------------------------------+--------+----------+----------+----------+
+| Parameter Name                | Type   | Default  | Min      | Max      |
++===============================+========+==========+==========+==========+
++-------------------------------+--------+----------+----------+----------+
+| max_query_cache_entries       | int    | 200      | 1        | INT_MAX  |
++-------------------------------+--------+----------+----------+----------+
+| max_query_cache_pages         | int    | 1,000    | 1        | INT_MAX  |
++-------------------------------+--------+----------+----------+----------+
+
+**max_query_cache_entries**
+
+    **max_query_cache_entries** is a parameter to configure the maximum number of query to be cached. If the **max_query_cache_entries** parameter is configured to 0 or a negative value, paramter configuration error occurs; if it is configured to an integer value equal to or greater than 1, a specified number of queries are cached with the result.
+
+    The following example shows how to cache up to 500 queries. ::
+
+        max_plan_cache_entries=500
+
+**query_cache_use_pages**
+
+    **query_cache_use_pages** is a parameter to configure the maximum page of result to be cached. If the **query_cache_use_pages** parameter is configured to 0 or a negative value, parameter configuration error occurs; if it is configured to an integer value equal to or greater than 1, specified pages in results are cached as temp files.
+
+    The following example shows how to cache up to 4,000 pages. ::
+
+        max_plan_cache_pages=4,000
```

Comment:
```suggestion
        max_plan_cache_pages=4000
```

Reply:
I modified code as follows:
query_cache_use_pages=4000

And modified code max_plan_cache_entries to max_query_cache_entries